api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1893,6 +1894,10 @@ func (address ProcessAddress) MarshalJSON() ([]byte, error) {
 // representation.
 func ParseProcessAddress(address string) (ProcessAddress, error) {
 	result := ProcessAddress{}
+	if strings.HasSuffix(address, "(fromHostname)") {
+		address = strings.TrimSuffix(address, "(fromHostname)")
+		result.FromHostname = true
```

Comment:
Do we really need this flag or should we just try to autodetect if the address is a DNS name or an IP address?